src/components/oscal-utils/OSCALLinkUtils.js

```diff
@@ -32,3 +32,8 @@ export default function resolveLinkHref(
     )
   );
 }
+
+export function getURLMediaType(url) {
+  const fileExtension = new URL(url).pathname.split(".").pop();
+  return fileExtension.length <= 4 ? fileExtension.toUpperCase() : "Unknown";
+}
```

Comment:
I want to see a lot of tests for this. Things like:

```
getURLMediaType("https://easydynamics.com/favicon.ico");
getURLMediaType("https://github.com/EasyDynamics/oscal-react-library/pull/574/files");
getURLMediaType("https://raw.githubusercontent.com/EasyDynamics/oscal-react-library/develop/README.md");
getURLMediaType("https://csrc.nist.gov/projects/united-states-government-configuration-baseline");
getURLMediaType("https://doi.org/10.6028/NIST.SP.800-97");
```

And more. I want to see lots of tests to make sure we understand the expected behavior of this function.

Reply:
For clarity:

```ts
expect(getURLMediaType("https://easydynamics.com/favicon.ico")).toBe("ICO");
```

or whatever